### ISSUE TITLE
Evaluations Data

### ISSUE THREAD

**Issue by AdamMcAdamson** [UTDNebula/nebula-api#158]
We would like to provide evaluation data as part of our API.

To this effect, we need to:
- [ ] Extend our database schema to include Evalution data
- [ ] Create Golang model conforming to schema
- [ ] Develop a scraper for [chromedp](https://github.com/chromedp/chromedp) to scrape the data available from [https://coursebook.utdallas.edu/ues-report/](https://coursebook.utdallas.edu/ues-report/)

**Comment by kneevin** [comment]
hey! i'd like to take on this scraper.

**Comment by AdamMcAdamson** [comment]
That sound good to me, @hochladen?

Also, this issue should probably be moved to https://github.com/UTDNebula/api-tools.

**Comment by hochladen** [comment]
I'm open to it; though I can say that this could also be implemented as a part of the existing coursebook scraper (as well as anything else we may need to pull from coursebook)

I'm not entirely opposed to having this as a separate scraper, but I'd say it's a matter of considering if the separation of tasks would be worth the added clutter.

**Comment by AdamMcAdamson** [comment]
Right, I had forgotten that we can pull eval data from the coursebook scraper with the speedup.

**Comment by kneevin** [comment]
so should i try to refactor the current scraper so it can scrape the eval. data as well? or is it ok to have two separate scrapers? i think it'll make more sense to refactor the current scraper it'll just take me a bit more time to figure it out

**Comment by hochladen** [comment]
Adding it to the current scraper would probably be best, though I still have to push it since it's still WIP and stored on my PC locally at the moment. It'll probably be a day or two before I do that, since I'm currently in the middle of moving back home.

**Comment by jpahm** [comment]
Yeah so it's fair to say "a day or two" was a vast underestimate of how long this would take to add; regardless, it should be added soon as part of the existing scraper now that other priorities have been taken care of.
